plugins/by-name/barbar/default.nix

```diff
@@ -195,11 +195,6 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
     );
 
   extraOptions = {
-    iconsPackage = lib.mkPackageOption pkgs [
-      "vimPlugins"
-      "nvim-web-devicons"
-    ] { nullable = true; };
```

Comment:
Should we have removed-option-modules for these? We've only had the options for 3 weeks, but they are pretty user-facing :thinking: 

I'm not yet voicing an opinion, just raising the issue for discussion.

Reply:
That's what I was wondering, too...

Reply:
I might just do it to be safe... I know we said we didn't need to in the linked issue, but it took me longer to get to this than I expected.